.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: cpp
 sudo: false
+dist: trusty
```

Comment:
Travis has introduced Xenial in the past few weeks. I suggest using it with: `dist: xenial`.